Resources/public/Admin.js

```diff
@@ -342,9 +342,9 @@ var Admin = {
             var collection = jQuery(this);
             var counter = 0;
             collection.children().each(function() {
-                var matches = highestCounterRegexp.exec(jQuery(this).find('.form-control').attr('id'));
+                var matches = highestCounterRegexp.exec(jQuery('[id^="sonata-ba-field-container"]', this).attr('id'));
                 if (matches && matches[1] && matches[1] > counter) {
-                    counter = matches[1];
+                    counter = parseInt(matches[1]);
```

Comment:
No there is no default.  If matches is 05 it will be parse as Hexa. Learn
once this a long time ago...

On Sun, Apr 5, 2015, 14:23 Sullivan SENECHAL <notifications@github.com>
wrote:

> In Resources/public/Admin.js
> <https://github.com/sonata-project/SonataAdminBundle/pull/2889#discussion_r27776896>
> :
>
> >                  if (matches && matches[1] && matches[1] > counter) {
> > -                    counter = matches[1];
> > +                    counter = parseInt(matches[1]);
>
> Is 10 not the default of the second parameter?
>
> And why just matches? It's an array.
>
> Should be parseInt(matches[1], 10) instead, isn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonata-project/SonataAdminBundle/pull/2889/files#r27776896>
> .
>